src/test/java/NumberTest.java

```diff
@@ -0,0 +1,16 @@
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class NumberTest {
+
+    @Test
+    void createTest() {
+        Number number = new Number();
+        assertThat(number.create().chars()
+            .distinct()
+            .count()).isEqualTo(3);
+        assertThat(number.create().chars()
+            .noneMatch(numberCharacter -> numberCharacter <= 48 || numberCharacter > 57)).isTrue();
```

Comment:
이부분은 아래와 같이 리팩토링 해보면 어떨까 싶습니다. 어떤 점이 더 좋은지 고민해보셨으면 좋겠네요 : D 
```suggestion
            .noneMatch(numberCharacter -> '0' <= numberCharacter && numberCharacter <= '9');
```

또한 assertThat() 안의 파라미터를 변수로 빼내는건 어떨까요?

Reply:
리팩토링 코드의 제시 감사드립니다. 아스키코드 대신에 char 타입으로 바료 표기해주니 훨씬 더 이해하기가 쉬운 것 같습니다. 또, 앞서 말씀해주신 시나리오별 단위테스트 작성을 유념하여 수정하였고 변수도 밖으로 빼서 함수의 호출을 줄였습니다.

++질문입니다.  조심스럽게 여쭤봅니다만, 혹시 리팩토링으로 제시해주신 코드의 끝에 .isFalse()를 빠뜨리신건지 아니면 의도하신건지 궁금합니다. 
또, noneMatch()를 이용한다면 모든 요소가 1 ~ 9 사이의 숫자가 아닌가? 에 대한 검증인데  noneMatch로 numberCharacter가 1 ~ 9 사이의 숫자인가를 만족하는지 검증하게 되면 본질적으로 모든 요소가 숫자임을 검증하는 것이 목적이므로 목적성에 맞게 생각해봤을때 allMatch()를 사용하는게 좀 더 직관적이고 좋지 않을까 생각되는데 어떻게 생각하시나요? 그러면 이 경우에는 .isTrue() 가 적합하겠지요.